orca-queue/src/main/kotlin/com/netflix/spinnaker/orca/q/metrics/AtlasQueueMonitor.kt

```diff
@@ -71,6 +89,29 @@ class AtlasQueueMonitor
     _lastState.set(queue.readState())
   }
 
+  @Scheduled(fixedDelayString = "\${queue.zombie.check.frequency:3600000}")
```

Comment:
`frequency` vs `frequencyMs`

Reply:
We also have other examples in `orca.yml` where we're calling this `intervalMs` 🤷‍♂️